.circleci/config.yml

```diff
@@ -595,15 +596,12 @@ jobs:
           buildSteps:
             - "run":
                 name: "Unit Test"
+                environment:
+                  # Once dependencies are built, we can allow some more concurrency for our own
+                  # test suite.
+                  UNITTEST_CORES: 8
                 command: |
-                  source .circleci/lib.sh
```

Comment:
is this now unused? (Don't see any uses in this file). If so, please delete.

Reply:
Sorry, I'm not sure to understand your comment.
Indeed, `lib.sh` is deleted by this PR and will no longer referenced from this file (was mostly doing Cachix things).

Or do you mean something else?

Reply:
@meejah: are we still missing something here?